scripts/install

```diff
@@ -50,6 +50,29 @@ ynh_app_setting_set --app=$app --key=with_carddav --value=$with_carddav
 ynh_app_setting_set --app=$app --key=with_enigma --value=$with_enigma
 ynh_app_setting_set --app=$app --key=language --value=$language
 
+#=================================================
+# ADD SWAP IF NEEDED
+#=================================================
+
+total_memory=$(ynh_get_ram --total)
+total_swap=$(ynh_get_ram --total --only_swap)
+swap_needed=0
+
+if [ $total_memory -lt 4096 ]; then
+    # Need a minimum of 4GB of memory
+    swap_needed=$((4096 - $total_memory))
+fi
+
+# Need at least 2GB of swap
+if [ $(($total_swap + $swap_needed)) -lt 2048 ]; then
+    swap_needed=$((2048 - $total_swap))
+fi
```

Comment:
Out of curiosity, did you try using `ynh_require_ram` for this ? Or was it not matching what you aim to want to do ? Can you elaborate on this ? (So that maybe we can improve the official helper)